pyotgw/protocol.py

```diff
@@ -21,6 +21,7 @@
 import re
 import struct
 from asyncio.queues import QueueFull
+from queue import Empty
```

Comment:
get_nowait raises QueueEmpty from asyncio.queues

Reply:
Resolved in ac3c3ab